src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Issue Platform - Writing Detectors"
+---
+
+# Fingerprinting
+
+Error events are grouped based on a set of strategies like stack traces, etc. These strategies can be customized by users. Error fingerprinting is a complex ever-evolving mechanism that has been around for many years.
+
```

Comment:
I feel like maybe this isn't needed here. I added it to the doc to contrast error fingerprinting to performance issue fingerprinting but it doesn't seem necessary here